ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/adoptopenjdk_install/tasks/main.yml

```
@@ -157,13 +157,32 @@
   when:
     - ansible_distribution == "MacOSX"
     - adoptopenjdk_installed.rc != 0
+    - jdk_version != 10
   tags: adoptopenjdk_install
 
 - name: Run installer (macOS)
   shell: sudo installer -pkg /tmp/{{ jdk_version }}-installer.pkg -target /
   when:
     - ansible_distribution == "MacOSX"
     - adoptopenjdk_installed.rc != 0
+    - jdk_version != 10
+  tags: adoptopenjdk_install
+
+- name: Download binary when installer is not available (MacOS)
```

Can we change `binary` to tarball, since the installer is technically a binary too.
```suggestion
- name: Download tarball when installer is not available (MacOS)
```